### PR TITLE
Make scale methods per trigger internal

### PR DESCRIPTION
**Pull request by alrod**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by JoshLove-msft — APPROVED** [review]

File: sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/src/Microsoft.Azure.WebJobs.Extensions.Storage.Queues.csproj [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <Version>5.2.0-beta.1</Version>
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
     <ApiCompatVersion>5.1.3</ApiCompatVersion>
+		<RunApiCompat>false</RunApiCompat>
```
Comment (JoshLove-msft) [inline sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/src/Microsoft.Azure.WebJobs.Extensions.Storage.Queues.csproj:9]: Why was this changed? If the method has already shipped we can't remove it.

Comment (alrod) [reply sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/src/Microsoft.Azure.WebJobs.Extensions.Storage.Queues.csproj:9]: That's what was done for Microsoft.Azure.WebJobs.Extensions.Storage.Blobs in https://github.com/Azure/azure-sdk-for-net/commit/5a5adff5193a03d8f1b731ffa194609278b39e5c - the public method was removed.

Microsoft.Azure.WebJobs.Extensions.Storage.Blobs 5.1.3 - public static IWebJobsBuilder AddAzureStorageBlobsScaleForTrigger
Microsoft.Azure.WebJobs.Extensions.Storage.Blobs 5.2.0 - internal static IWebJobsBuilder AddAzureStorageBlobsScaleForTrigger

Comment (JoshLove-msft) [reply sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/src/Microsoft.Azure.WebJobs.Extensions.Storage.Queues.csproj:9]: That shouldn't be necessary, please remove it.

Comment (mathewc) [reply sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/src/Microsoft.Azure.WebJobs.Extensions.Storage.Queues.csproj:9]: Yes, I'm confused as well. All the discussions we've been having on the naming of these methods and their visibility led me to believe they haven't been released publicly yet.

Comment (alrod) [reply sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/src/Microsoft.Azure.WebJobs.Extensions.Storage.Queues.csproj]: I had realized that we already released the method for storage extensions as latest blob extension has the method as internal. I scoped the PR to EH and SB only.

**Review by JoshLove-msft — APPROVED** [review]

**Review by mathewc — APPROVED** [review]
